ext/standard/file.c

```diff
@@ -1467,6 +1467,34 @@ PHP_FUNCTION(unlink)
 }
 /* }}} */
 
+/* {{{ Sync file to storage. Similar to fflush() but blocks until OS buffers have flushed. */
+PHP_FUNCTION(fsync)
+{
+	zval *res;
+	int ret;
+	php_stream *stream;
+
```

Comment:
PHP source is indented with tabs, not spaces.

Reply:
Oof, didn't occur to me and my IDE is set to replace tabs with 4 spaces automatically....will update all changed files accordingly, cheers.

Reply:
No, wait....it isn't set to do that. Weirdly in my IDE this is showing as tab indented and definitely with a tab character rather than spaces.

Reply:
See https://github.com/php/php-src/pull/6650#discussion_r567274283 is appearing tab spaced for me?